flake.nix

```diff
@@ -17,6 +17,7 @@
             jq
             just
             pnpm
+            nodejs_22
```

Comment:
This change just ensures a compatible version of nodejs is included in the nix dev env, which I use to manage the tooling locally.